frontend/src/css/url-builder.css

```
@@ -25,44 +25,44 @@ body#url-builder {
 
 #url-result--url {
     padding: 0.75rem;
-
-    background: rgba(51, 51, 51, 0.05);
-    border: 1px solid var(--mid-grey);
-    border-radius: 0.375rem;
+    font-size: 1rem;
 
     font-weight: 700;
```

Stylelint (17.5.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)